graphql_api/types/repository/repository.py

```diff
@@ -283,6 +283,15 @@ def resolve_flags_measurements_backfilled(repository: Repository, info) -> bool:
     return dataset.is_backfilled()
 
 
+@repository_bindable.field("isATSConfigured")
+def resolve_is_ats_configured(repository: Repository, info) -> bool:
+    if not repository.yaml or "flag_management" not in repository.yaml:
+        return False
+
+    individual_flags = repository.yaml["flag_management"].get("individual_flags", {})
+    return individual_flags.get("carryforward_mode") == "labels"
```

Comment:
Last but not least, I'd give a bit of an explanation why labels = ats configured. Will come in handy when someone, like myself, that doesn't know much about ATS sees this and needs the context for why this leads to ATS you know